Userland/Libraries/LibGfx/ImageFormats/BilevelImage.cpp

```diff
@@ -32,71 +32,94 @@ void BilevelImage::fill(bool b)
         byte = fill_byte;
 }
 
-template<BilevelImage::CompositionType operator_>
-void BilevelImage::composite_onto(BilevelImage& out, IntPoint position) const
+template<OneOf<BilevelImage, BilevelSubImage> InputType, BilevelImage::CompositionType operator_>
```

Comment:
commit description: "this makes the time
spent in `BilevelImage::subbitmap` drops from 26% to 0.03%." s/drops/drop/